doc/user_manual/chSpecialElements.tex

```diff
@@ -1520,19 +1521,22 @@ \section{Scattering} \label{sec:scatter}
 
 \bigskip
 
-\paragraph{DEBUG} \texttt{DEBUG}\\
+\paragraph{Module Debugging}~ \texttt{DEBUG}\\
 
 This statement switches on extra ``debugging'' output from \texttt{SCATTER}.
 This can be useful if debugging the code or if debugging the input.
 
-\paragraph{ELEMent} \texttt{ELEM elemname profile scaling gen1 (gen2, (gen3))}\\
+\paragraph{Scatter Elements}~ \texttt{ELEM elemname profile ratio scaling gen1 ... genN}\\
 
-This statements associates a \texttt{PRO}file and between one and 3\footnote{Controlled by the parameter \texttt{scatter\_maxGenELEM}.} \texttt{GEN}erators with a SINGLE ELEMENT which must be of type 40, as described in Section~\ref{SCAT}.
-The \texttt{scaling} argument, which is a floating point number, is used to scale the probability of an interaction.
-This can be controlled through DYNK, for example in order to scale only at one specified turn.
+This statements associates a \texttt{PRO}file and one or more \texttt{GEN}erators with a SINGLE ELEMENT.
+The element must be of type 40, as described in Section~\ref{SCAT}.
+The \texttt{ratio} argument (string/float) can be used to set a fixed scattering probability at the element.
+Alternatively, the field can be set to ``auto'', in which case the scattering probability is calculated from the cross section and density profile.
```

Comment:
Can you set `auto` from DYNK?

Reply:
`auto` is for ratio, not scaling. So no.

Reply:
Sorry, my bad. It's pretty clear as-is.